### DISCUSSION TITLE
Spotube plugin for Slimserveur (Logitech Media Server / Squeezebox)

### DISCUSSION THREAD

**Discussion by GG700** [KRTirtho/spotube#1432] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

Is that possible to develop a plugin SPOTUBE for Logitech Media Server ?
It is a music server free (Github : https://github.com/LMS-Community/slimserver).

Deezer and Spotify plugins are available in LMS but not Spotube. 

### Suggested solution

.

### Useful resources

.

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Sorry but we do not create plugins for 3rd party application. But if the community wants they can create one.
Starting from 3.6.0, Spotube exposes a WebSocket API that can discovered and used to control the playback. So a custom plugin could use this API.

Also, Spotube doesn't store the playback in any backend server. It purely runs locally. So I'm not sure if it's possible
